src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/Properties/launchSettings.json

```diff
@@ -21,7 +21,7 @@
       "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
-        "ASPNETCORE_URLS": "https://localhost:44300;http://localhost:8080"
+        "ASPNETCORE_URLS": "https://localhost:5000;http://localhost:5001"
```

Comment:
If these ports are typically used for other purposes(on different OSes), then we might want to change them to something else.

Reply:
These are not reserved ports. They're commonly used by various web frameworks for dev.